corehq/apps/translations/static/translations/js/setup.js

```diff
@@ -0,0 +1,11 @@
+hqDefine('translations/js/setup', [
```

Comment:
Hooray for files being requirejs-compatible right out of the gate.

FWIW, since this file is only doing select2 initialization, instead of creating this file you could add the class `ko-select2` to the two inputs and then use `hqwebapp/js/widgets` as `requirejs_main`.

Reply:
my first requirejs-compatible js contribution removed thought.. 
some day..

Reply:
There are a lot of migration tasks if you would like one ;)